src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js

```diff
@@ -32,8 +32,10 @@ import {
 import {
   isOngoing,
 } from '../../../common/POFields';
+import { WORKFLOW_STATUS } from '../../../common/constants';
 import { ItemForm } from '../../../components/POLine/Item';
 import { CostForm } from '../../../components/POLine/Cost';
+
```

Comment:
I'm not sure if we have this empty line in s code styleguide

Reply:
it's to separate inner and outer resources